payment_paypal/indico_payment_paypal/plugin.py

```diff
@@ -61,13 +72,27 @@ def get_blueprints(self):
 
     def adjust_payment_form_data(self, data):
         event = data['event']
+        event_settings_form = data['event_settings']
         registration = data['registration']
         plain_name = str_to_ascii(remove_accents(registration.full_name))
         plain_title = str_to_ascii(remove_accents(event.title))
         data['item_name'] = f'{plain_name}: registration for {plain_title}'
         data['return_url'] = url_for_plugin('payment_paypal.success', registration.locator.uuid, _external=True)
         data['cancel_url'] = url_for_plugin('payment_paypal.cancel', registration.locator.uuid, _external=True)
         data['notify_url'] = url_for_plugin('payment_paypal.notify', registration.locator.uuid, _external=True)
+        # Add Paypal fees
+        amount = float(data['amount'])
+        data['paypal_amount'] = amount
+        paypal_fixed_fee = float(event_settings_form['paypal_fixed_fee'])
+        paypal_percent_fee = float(re.findall(r'^([0-9]+(\.[0-9]+)?)(%)?', event_settings_form['paypal_percent_fee'])[0][0])
+        amount_with_paypal_fees = round (( amount + paypal_fixed_fee ) / ( 1 - ( paypal_percent_fee / 100 )), 2)
+        data['amount'] = amount_with_paypal_fees
+        fees = amount_with_paypal_fees - amount
+        if fees > 0.0:
+            data['paypal_fees'] = str(amount_with_paypal_fees - amount)
+        else:
+            data['paypal_fees'] = ''
```

Comment:
If these are stored as decimals you can just:
```suggestion
        data['paypal_fees'] = str(amount_with_paypal_fees - amount)
```

Reply:
done